cookiecutter.json

```diff
@@ -1,7 +1,7 @@
 {
 	"author_name": "Elixir Cloud AAI",
```

Comment:
Please ensure that all mentions of ELIXIR are fully capitalized.

Reply:
Can you check where do I have to cahnge it, because in test proj, pyproject.toml, I have used `Elixir`. 

Please do a global search or grep inside this project for Elixir.

Reply:
I don't know what you mean. Why don't you just do the search for `Elixir` and replace with `ELIXIR`? As you say, I think it's at least here, in the Cruft config, `pyproject.toml` and the `LICENSE`.